spec/grape-swagger/entities/response_model_spec.rb

```diff
@@ -191,6 +201,7 @@ class SomeEntity < Grape::Entity
           expose :nested, using: TheseApi::Entities::Nested, documentation: { desc: 'Nested object.' }
           expose :nested_child, using: TheseApi::Entities::NestedChild, documentation: { desc: 'Nested child object.' }
           expose :polymorphic, using: TheseApi::Entities::Polymorphic, documentation: { desc: 'Polymorphic Model' }
+          expose :mixed, using: TheseApi::Entities::MixedType, documentation: { desc: 'A model with mix of types' }
```

Comment:
```suggestion
          expose :mixed, using: TheseApi::Entities::MixedType, documentation: { desc: 'A model with mix of types.' }
```

(and let's replace Polymorphic Model with `A polymorphic model.` above, to be consistent?

Reply:
done here: [47e6140](https://github.com/ruby-grape/grape-swagger-entity/pull/75/commits/47e614066d3eacfb6f6ee559297a0cb4485bf5cb)